frontend/src/scenes/project/Settings/DataAttributes.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button, Input, Skeleton } from 'antd'
+import { useActions, useValues } from 'kea'
+import React, { useState } from 'react'
+import { teamLogic } from 'scenes/teamLogic'
+
+export function DataAttributes(): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+    const { patchCurrentTeam } = useActions(teamLogic)
+    const [value, setValue] = useState(currentTeam?.data_attributes?.join(', ') || '')
+
+    if (!currentTeam) {
+        return <Skeleton paragraph={{ rows: 0 }} active />
+    }
+
+    return (
+        <>
+            <p>
+                Specify a comma-separated list of{' '}
+                <a
+                    href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes"
+                    rel="noreferrer noopener"
+                >
+                    data attributes
+                </a>{' '}
+                used in your app. For example: <code>data-attr, data-custom-id, data-myref</code>. These attributes will
+                be used when creating actions to match unique elements on your pages.
```

Comment:
Suggestion: `be used when using the toolbar and defining actions to match unique elements on your pages`